swap/swap.go

```diff
@@ -466,56 +303,56 @@ func (s *Swap) Balances() (map[enode.ID]int64, error) {
 	return balances, nil
 }
 
-// loadLastSentCheque loads the last cheque for a peer from the state store (persisted)
-// To be called with mutex already held
-// Caller must be careful that the same cheque isn't concurrently read and written by multiple routines
-func (s *Swap) loadLastSentCheque(peer enode.ID) (err error) {
-	//only load if the current instance doesn't already have this peer's
-	//last cheque in memory
+// loadLastReceivedCheque loads the last received cheque for the peer from the store
```

Comment:
Add: 
`and returns nil when there is no cheque ever saved`

Reply:
done, slightly different and added it to all the load functions